c2corg_ui/templates/xreport/view.html

```diff
@@ -137,7 +137,8 @@ <h3 class="heading informations" ng-click="mainCtrl.animateHeaderIcon($event)" d
 
   ## ASSOCIATIONS
   % if not version:
-    <div class="view-details-associations col-xs-12 associations">
+    <div class="view-details-associations col-xs-12 associations" ng-init="a = detailsCtrl.documentService.document.associations"
+         ng-show="a.waypoints.length || a.routes.length || a.articles.length || a.outings.length">
       <span class="lead">
 ##         <div ng-show="userCtrl.auth.isAuthenticated()" class="add-association">
```

Comment:
@ginold I deleted/added some code in https://github.com/c2corg/v6_ui/pull/1108 ... you can leave it now so I do not have to solve many conflicts during rebase later